test/e2e/flowaggregator_test.go

```diff
@@ -574,6 +564,26 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	assert.GreaterOrEqualf(t, dataRecordsCount, expectedNumDataRecords, "IPFIX collector should receive expected number of flow records. Considered records: ", len(recordSlices))
 }
 
+func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcPodName string, srcIP string, dstIP string) {
+	timeStart := time.Now()
+	cmdStr := fmt.Sprintf("curl %s", externalServer)
+	data.runCommandFromPod(testNamespace, srcPodName, nginxContainerName, []string{"/bin/sh", "-c", cmdStr})
```

Comment:
why do we need a shell instead of running curl directly?

shouldn't we check that the command is successfull?

Reply:
Yes, we should run it directly.
Added the check. Thanks!